plugins/org.eclipse.osee.framework.core/src/org/eclipse/osee/framework/core/enums/CoreBranchCategoryTokens.java

```diff
@@ -26,8 +26,9 @@ public final class CoreBranchCategoryTokens {
    public static final BranchCategoryToken ATS = new BranchCategoryToken(2L, "ATS");
    public static final BranchCategoryToken MIM = new BranchCategoryToken(3L, "MIM");
    public static final BranchCategoryToken CI = new BranchCategoryToken(4L, "CI");
+   public static final BranchCategoryToken PR = new BranchCategoryToken(5L, "PR");
 
-   public static List<BranchCategoryToken> values = Arrays.asList(PLE, ATS, MIM, CI);
+   public static List<BranchCategoryToken> values = Arrays.asList(PLE, ATS, MIM, CI, PR);
```

Comment:
Do we want to make this more specific? MIM_PR?

Reply:
Not sure... the category could be used for some other kind of peer review

Reply:
The point I'm making is that our UI wouldn't be able to handle those types of peer reviews...